src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -3660,6 +3699,7 @@ operand rRegL()
   constraint(ALLOC_IN_RC(long_reg));
   match(RegL);
   match(rax_RegL);
+  match(rcx_RegL);
```

Comment:
Why we need this change?

Reply:
Mostly for completeness only, do you think I should revert this? Thank you very much.

Reply:
> Mostly for completeness only, do you think I should revert this? Thank you very much.

Yes.
I would suggest not touching unnecessary part of code.

Reply:
Thanks, I have removed irrelevant changes.